packages/http-client-python/generator/pygen/codegen/serializers/__init__.py

```diff
@@ -153,8 +153,8 @@ def serialize(self) -> None:
             elif client_namespace_type.clients:
                 # add clients folder if there are clients in this namespace
                 self._serialize_client_and_config_files(client_namespace, client_namespace_type.clients, env)
-            else:
-                # add pkgutil init file if no clients in this namespace
+            elif not self.read_file(exec_path / Path("__init__.py")):
+                # add pkgutil init file if no clients in this namespace and no existing __init__.py file
```

Comment:
I thought about other solutions like add judge logic about whether namespace contains `_generated`. But we can't assume which kind of namespace users may set. Maybe they want namespace like `azure.test.generated` without `_` then the judge logic can't work again.

So the best solution is we don't overwrite the existing pkgutil init file: 
- If  `__init__` is generated by codegen, no need to overwrite it with same content
- If `__init__` is written by users, leave it unchanged.